src/_style.scss

```diff
@@ -329,3 +346,84 @@ $apple-focus-black: $apple-black;
     }
   }
 }
+
+.large-screen-background {
+  background-color: $white;
+  background-image: -webkit-linear-gradient(-11deg, $primary 50%, $white 50%);
+  width: 100%;
+}
+
+.medium-screen-background {
+  background-color: $white;
+  background-image: -webkit-linear-gradient(-11deg, $primary 93%, $white 7%);
```

Comment:
Same here, stick to one function for consistency. Use linear-gradient() and use the appropriate angle for it.